engine/class_modules/sc_shaman.cpp

```diff
@@ -5322,6 +5335,39 @@ struct thundercharge_t : public shaman_spell_t
   }
 };
 
+// ==========================================================================
+// Fae Transfusion - Night Fae Covenant
+// ==========================================================================
+
+struct fae_transfusion_tick_t : public shaman_spell_t
+{
+  fae_transfusion_tick_t( util::string_view n, shaman_t* player )
+    : shaman_spell_t( n, player, player->find_spell( 328928 ) )
+  {
+    affected_by_master_of_the_elements = false;
+    aoe        = 4;
+    background = true;
+    callbacks  = false;
+  }
+
+  result_amount_type amount_type( const action_state_t*, bool ) const override
+  {
+    return result_amount_type::DMG_DIRECT;
+  }
+};
+
+struct fae_transfusion_t : public shaman_spell_t
+{
+  fae_transfusion_t( shaman_t* player, const std::string& options_str )
+    : shaman_spell_t( "fae_transfusion", player, player->find_covenant_spell( "Fae Transfusion" ), options_str )
+  {
+    parse_options( options_str );
```

Comment:
Options are currently parsed twice for this spell, only one is necessary.

Reply:
This should be fixed in 3e1c4e2884